bin/cron/process_forms

```diff
@@ -146,7 +146,7 @@ def main
   receipt_count = send_batch_of_receipts(limit)
   index_batch_of_forms(limit)
 
-  system "#{deploy_dir('bin', 'deliver_poste_messages').shellescape} 2>&1" unless receipt_count == 0
+  system "#{deploy_dir('bin/cron', 'deliver_poste_messages').shellescape} 2>&1" unless receipt_count == 0
```

Comment:
I noticed this testing the email on localhost: 
```
sh: /Users/andrew/Code/code-dot-org/bin/deliver_poste_messages: No such file or directory
```

Long ago `deliver_poste_messages` was moved into the `cron` dir. This didn't generate a honeybadger error because of the `2>&1` piping both stderr and stdout to stdout, and no error code returned. It also didn't really matter because `deliver_poste_messages` runs every minute by cronjob so the emails still went out.